src/components/table/table.tsx

```diff
@@ -200,10 +200,12 @@ export class Table {
     private firstRequest: boolean;
     private currentSorting: ColumnSorter[];
     private tableSelection: TableSelection;
+    private shouldSort = false;
 
     constructor() {
         this.handleDataSorting = this.handleDataSorting.bind(this);
         this.handlePageLoaded = this.handlePageLoaded.bind(this);
+        this.handleRenderComplete = this.handleRenderComplete.bind(this);
```

Comment:
I think it is better to remove all these `bind` stuff in the cool down and convert those function to arrow functions instead.

Reply:
Yeah I had the same thought